cnfformula/__init__.py

```diff
@@ -29,7 +29,7 @@ def _load_formula_generators():
     # Load the formula generators in the `cnfformula` namespace
     self_ref = sys.modules[__name__]
     self_ref.__dict__.update(loot)
-    __all__.extend(name for name in loot.keys() if name not in __all__)
+    __all__.extend(name for name in list(loot.keys()) if name not in __all__)
```

Comment:
No need for `list()` here.

Reply:
Indeed, it was done automatically by the module [2to3](https://docs.python.org/3/library/2to3.html). Note that in python 2, keys returns a list whereas iterkeys returns an iterator.